tests/includes/juju.sh

```diff
@@ -188,15 +218,31 @@ juju_bootstrap() {
 		;;
 	esac
 
+	image_metadata=""
+
+	if [[ ${BOOTSTRAP_PROVIDER} == "vsphere" ]]; then
+		echo "====> Creating image simplestream metadata for juju ($(green "${version}:${cloud}"))"
+
+	  image_streams_dir=/tmp/image-streams
```

Comment:
I'd rather keep everything in `${TEST_DIR}`, so this would become `"${TEST_DIR}"/image-streams`. We just need to ensure that we make the directory first.